index.html

```diff
@@ -18,6 +18,7 @@
       <img src="./router.png" >
       <div class="pane-wrapper">
         <div id="model-pane" class="pane">
+          <span class="recommendedCheckBox"><input type="checkbox" id="recommendedselect" value="recommendedOnly" checked /><label for="recommendedOnly">Nur empfohlene Geräte anzeigen</label></span>
```

Comment:
Visibility of this checkbox should be configurable in the config-file for communities who want to completely remove the ability to download such firmware.